### ISSUE TITLE
valPSNR increase very slowly

### ISSUE THREAD

**Issue by hogeman2** [avinashpaliwal/Super-SloMo#21]
Nice work thanks!

I have a question about convergence speed

I've run your training code and almost passed 50 epoch.  The valPSNR is still under 15.5.

Is that normal?

also created clips seems to be not sequencial is that ok?

**Comment by avinashpaliwal** [comment]
- I have added [tensorboard](https://github.com/avinashpaliwal/Super-SloMo#tensorboard). How do the interpolated images look on the validation set?

- Are you using a custom dataset or adobe240fps dataset?
   - If you are using a custom dataset, then it should have slow motion videos. The [dataloader ](https://github.com/avinashpaliwal/Super-SloMo/blob/e8508bd949e51985ba73b99d0063090cec7e3679/dataloader.py#L186)takes frame1 and frame8 as reference and interpolates the frames in between.

- Did you use the [create_dataset](https://github.com/avinashpaliwal/Super-SloMo/blob/master/data/create_dataset.py) script?

**Comment by hogeman2** [comment]
Thanks for your reply
First interpolated images looks not good and blurred.

And I used adobe240dataset using create_dataset script

**Comment by avinashpaliwal** [comment]
Can you attach the interpolated results from tensorboard? Also specify your setup- CPU/GPU, PyTorch version, OS, etc. 15.5 valPSNR is really low. Maybe there is a problem with dataset or setup.

**Comment by hogeman2** [comment]
I'm using GTX 1060
Is that problem reducing batch size to 4?

![screenshot from 2019-01-16 11-50-48](https://user-images.githubusercontent.com/41729977/51223476-4033eb80-1985-11e9-8d99-a0a28d247489.png)
![screenshot from 2019-01-16 11-51-37](https://user-images.githubusercontent.com/41729977/51223478-41fdaf00-1985-11e9-9871-8c322cc89fcb.png)

I added validation image, and graph

**Comment by hogeman2** [comment]
https://github.com/avinashpaliwal/Super-SloMo/issues/22 
seems like this was the problem
I've changed code and graph looks good.

**Comment by avinashpaliwal** [comment]
> #22
> seems like this was the problem
> I've changed code and graph looks good.

Right. The two reference (left and right corner) look very different. Frames in a clip look very similar since they are slow motion videos. Where exactly in code did you find the bug?

**Comment by hogeman2** [comment]
@avinashpaliwal line 69 in create_dataset.py. 
I changed line
images = os.listdir(os.path.join(root, file))
to 
images = sorted(os.listdir(os.path.join(root, file)))

**Comment by shliang0603** [comment]
@avinashpaliwal  I have the same problem. I had  trained the model to 270 epoch, but the valPSNR  keep bellow 15, can you help me ?
![image](https://user-images.githubusercontent.com/37186266/53713665-9c47c600-3e86-11e9-891c-f420bdfe50b6.png)

**Comment by avinashpaliwal** [comment]
> @avinashpaliwal I have the same problem. I had trained the model to 270 epoch, but the valPSNR keep bellow 15, can you help me ?
> ![image](https://user-images.githubusercontent.com/37186266/53713665-9c47c600-3e86-11e9-891c-f420bdfe50b6.png)
> 
> the follow is valPSNR changing curve
> ![image](https://user-images.githubusercontent.com/37186266/53715285-483fe000-3e8c-11e9-9280-ac669b7c0421.png)

Try the new create_dataset script to generate the dataset and try training again. The model starts converging right from the beginning.
